src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -8951,6 +9007,10 @@ private MethodSymbol CheckMethodGroupReceiverNullability(BoundMethodGroup group,
         private void VisitLambda(BoundLambda node, NamedTypeSymbol? delegateTypeOpt, Optional<LocalState> initialState = default)
         {
             Debug.Assert(delegateTypeOpt?.IsDelegateType() != false);
+#if DEBUG
```

Comment:
The preprocessing directive isn't necessary.

Reply:
I think it is needed. Even though the `Debug.Assert` call is not emitted in Release config, it's still bound by the compiler, so it would complain that `_errorLambdas` doesn't exist.

Reply:
Ah, missed that. Thanks